pkg/brctl/brctl.go

```diff
@@ -188,7 +193,7 @@ func Show(out io.Writer, names ...string) error {
 
 		for _, bridge := range devices {
 			// check if device is bridge, aka if it has a bridge directory
-			_, err := os.Stat(BRCTL_SYS_NET + bridge.Name() + "/bridge/")
+			_, err := os.Stat(BRCTL_SYS_NET + "/" + bridge.Name() + "/bridge/")
```

Comment:
To not get issues like this you could also use `filepath.Join()` as it will make sure the slashes are placed correctly.